content/04.results.md

```diff
@@ -2,15 +2,15 @@
 Our main goal is to test the performance of methods to identify features that discriminate between groups and optimize the classification accuracy.
 
 ### Simulated data
-We compare the accuracy of each method for *r* = 100 replicate simulated data sets with moderate interaction effect. 
-These values of the effect size in the simulations generate adequately challenging data sets so that the methods' accuracies stay moderate and do not cluster around 0.5 or 1. 
-Each replicate data set is split into training and holdout. 
+We compare the accuracy of each method for *r* = 100 replicate simulated data sets with moderate interaction effect.
+These values of the effect size in the simulations generate adequately challenging data sets so that the methods' accuracies stay moderate and do not cluster around 0.5 or 1.
+Each replicate data set is split into training and holdout.
 The TPOT-DS, standard TPOT and XGBoost models are built from the training dataset, then the trained model is applied to the independent holdout data to obtain the generalization accuracy.
 
 Our simulation design produces a reasonable distribution of the functional features in all subsets, of which proportions are shown in Table [S1].
 According to Eq. {@eq:p_subset}, the earlier the subset, the more functional features it has.
-Therefore, our first aim is to determine how well TPOT-DS can identify the first subset 1 that contains the largest number of informative features. 
-With the specified template `Dataset Selector-Transformer-Classifier`, in 100 replications, TPOT-DS correctly selects subset 1 in the resulting pipeline 75 times (Fig. {@fig:simDS}), with an average cross-validated accuracy on the training set of 0.73 and out-of-sample accuracy of 0.69. 
+Therefore, our first aim is to determine how well TPOT-DS can identify the first subset 1 that contains the largest number of informative features.
+With the specified template `DatasetSelector-Transformer-Classifier`, in 100 replications, TPOT-DS correctly selects subset 1 in the resulting pipeline 75 times (Fig. {@fig:simDS}), with an average cross-validated accuracy on the training set of 0.73 and out-of-sample accuracy of 0.69. 
```

Comment:
I think we should leave space in `DatasetSelector` to stay consistent with previous text. I think it should be clear to the users if they read the TPOT github documentation.

Reply:
OK, agree